### PR TITLE
fix: missing updates for logger.ts

### PR DESCRIPTION
**Pull request by yodamaster726**
Somehow I missed this file update on my last PR.

- Adds a method for showing download progress used by ollama.ts when it downloads the llama local mode.
- Fixes the VERBOSE env variable for verbose logging

### CODE REVIEW

File: packages/core/src/defaultCharacter.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const defaultCharacter: Character = {
     username: "eliza",
     plugins: [],
     clients: [],
-    modelProvider: ModelProviderName.OPENAI,
+    modelProvider: ModelProviderName.OLLAMA,
```
[the remark this answers is not in the document]
Comment (yodamaster726) [reply packages/core/src/defaultCharacter.ts]: I'm updating it back to OPENAI as it was.

**Comment by yodamaster726** [comment]
> Great work - please remove the change in defaultCharacter and get the CI / CD pipeline working :)

working on it

**Comment by yodamaster726** [comment]
done

**Comment by yakul0** [comment]
ai16z is full of scammers do not work or contribute code to these people.

They have been pumping and dumping crypto coins for the past week on retail investors: $eliza, $matr, $vs and more. 

https://dexscreener.com/solana/2pofqcvn4wysazpsqppxq4cspbtkqlsehf9q5x77avm7
https://dexscreener.com/solana/hzqcylo7gjl5xvvglc2zfgfuwzkunhrzvge2hn9sysbr

scammers:
https://x.com/mattyryze
https://x.com/shawmakesmagic/status/1857903111073067315
https://x.com/pindarosothecat

Bunch of people calling out the ai16z team: 
https://x.com/spacepixel/status/1860511246241661274
https://x.com/JW100x/status/1860501669907189772
https://x.com/0xApeRich/status/1860492861357060381
https://x.com/BullTrx1/status/1860492092767695108

DYO. Don't help work for free for these people. Fork the repo and contribute somewhere else.

**Comment by yodamaster726** [comment]
@monilpat I've made the requested fixes/updates.

**Comment by jkbrooks** [comment]
minor conflict @yodamaster726 @monilpat `pnpm-lock.yaml`

**Pull request merged**